### PR TITLE
Use Vector2 for well-related visualizations - Part 1

### PR DESCRIPTION
**Pull request by andreabrambilla**
Supersedes PR #330 

This is a first small step in using Vector2 instead of the home made vector algebra for anything that has to be displayed along the wellbore (hole, cement, casing, etc). We have just started using Vector2 in vectorUtils.ts and we are just reproducing the previous behavior. Further refactoring and optimization will come in later PRs

### CODE REVIEW

**Comment by andreabrambilla** [comment]
@stalenestas I started from your PR, rebased it on top of master and just made changes to vectorUtils.ts. Please have a look at all the changes outside of vectorUtils.ts to see if I forgot something while rebasing, or if some of the changes are not relevant anymore

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ dist
 .storybook/src/exampledata/**
 storybook-static
 report*
+.storybook/src/api/index.ts
```
Comment (stalenestas) [inline .gitignore:6]: Not sure if this needs to be checked in.

**Comment by andreabrambilla** [comment]
@Brynjulf @HavardNJ can one of you have a quick look at this please?

**Comment by andreabrambilla** [comment]
Thanks for the review and the cleanup 👍

**Pull request merged**